tasks/configure.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Defines which parameters/files are editable by owners
```

Comment:
The task names specify what is being done in a task, like `Template edit_list configuration`.
The same thing with the other task names.